client/components/domains/register-domain-step/index.jsx

```diff
@@ -525,10 +526,12 @@ const RegisterDomainStep = React.createClass( {
 	showValidationErrorMessage: function( domain, error ) {
 		let message,
 			severity = 'error';
+		const tld = domain.lastIndexOf( '.' ) !== -1 && domain.substring( domain.lastIndexOf( '.' ) ),
```

Comment:
You could probably save `domain.lastIndexOf( '.' )` and re-use it, but that's up to you.